tests/test_spike_locations.py

```diff
@@ -0,0 +1,71 @@
+# 8/23/22
+# https://figurl.org/f?v=gs://figurl/spikesortingview-8&d=sha1://d8ed07c1d4653f0e22c80471f4c921c4aa7d30fc&label=test_spike_locations
+
+import numpy as np
+from typing import List
+import sortingview as sv
+import sortingview.views as vv
+import spikeinterface as si
+import spikeinterface.extractors as se
+
+
+def main():
+    recording, sorting = se.toy_example(num_units=12, num_channels=10, duration=300, seed=0)
+
+    R = sv.copy_recording_extractor(recording, serialize_dtype='float32')
+    S = sv.copy_sorting_extractor(sorting)
+
+    view = test_spike_locations(recording=R, sorting=S, hide_unit_selector=False)
+
+    url = view.url(label='test_spike_locations')
+    print(url)
+
+def test_spike_locations(*, recording: si.BaseRecording, sorting: si.BaseSorting, hide_unit_selector: bool=False):
+    channel_locations = recording.get_channel_locations().astype(np.float32)
+    xmin = np.min(channel_locations[:, 0])
+    xmax = np.max(channel_locations[:, 0])
+    ymin = np.min(channel_locations[:, 1])
+    ymax = np.max(channel_locations[:, 1])
+    xspan = xmax - xmin
+    yspan = ymax - ymin
+    if xmax <= xmin:
+        xmin = xmin - 12
+        xmax = xmax + 12
+    if ymax <= ymin:
+        ymin = ymin - 12
+        ymax = ymax + 12
```

Comment:
This is weird--why would this happen?

Reply:
This happens if all the electrodes have the same x (or y) coordinate